apps/src/music/appConfig.js

```diff
@@ -21,8 +23,16 @@ export const getBlockMode = () => {
   return blockMode;
 };
 
+export function setAppConfig(values) {
```

Comment:
should the app config be moved into redux? It looks like a mini state store to me, is that effectively what it is?

Reply:
It's essentially a convenient way to query a static configuration for the app instance, and none of the values change once Music Lab is loaded, even when changing levels.  The values can be perused at https://studio.code.org/musiclab/menu, but as we progress further in development, we use these less and less.  

I think we could consider moving these to a store, but I think it might be better to leave that for a potential follow-up, to keep this change better scoped.

Reply:
I agree this can be done as a potential follow-up. The reason I brought it up here is we now have a `setAppConfig` method that gets called from within `MusicView`, so it looked to me like the values could change, but is that not the case?
If they can't change, then maybe react context is a reasonable place for them? If they are only going to be used in development, and will eventually go away we can also leave as-is.

Reply:
> I agree this can be done as a potential follow-up. The reason I brought it up here is we now have a `setAppConfig` method that gets called from within `MusicView`, so it looked to me like the values could change, but is that not the case?

Oh, those values are passed in when the lab is created on a script-level or level page, but shouldn't change dynamically, [here](https://github.com/code-dot-org/code-dot-org/pull/50923/files#diff-37d3d3d7e40a2e6cab18609ec431f26866a17a710efe34daa6e38bd1bcb3c1dcR10-R23).